include/picongpu/plugins/openPMD/Json_private.hpp

```diff
@@ -268,6 +268,6 @@ namespace picongpu
             std::string getDefault() const override;
         };
 
-        std::vector<std::string> const JsonMatcher::m_recognizedBackends = {"adios1", "adios2", "hdf5", "json"};
+        std::vector<std::string> const JsonMatcher::m_recognizedBackends = {"adios2", "hdf5", "json"};
```

Comment:
If we do not provide adios2, openPMD-api could still use adios(1), correctly?
Why would you thus remove this `"adios1"` json tag?

Reply:
Yes, that is a valid question. I thought this piece is only for PIConGPU to parse arguments given as json, and so adios1 should be removed. However, i am not 100% sure. @franzpoeschel do you happen to know it?

Reply:
Simple answer: I wrote this with the plan in mind that this could one day be mainlined into openPMD-api (listing the `json` backends doesn't make a lot of sense either)As . If a user defines a configuration for backends ADIOS1 or JSON, it will be handled by the regex matcher, but the resulting config will never be chosen.
We could alternatively just initialize this vector via `::openPMD::getVariants()`

Reply:
> If we do not provide adios2, openPMD-api could still use adios(1), correctly?

No, ADIOS1 is deactivated in the openPMD plugin. If PIConGPU detects that the ADIOS1 backend has been selected, it throws an error.

Reply:
Thanks for the last explanation @franzpoeschel  - that leads me to the following question ...

Reply:
So is my change of this piece fine, or it should be `::openPMD::getVariants()`, or it doesn't matter for us?

Reply:
I guess it doesn't really matter for PIConGPU